zulip_bots/setup.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import sys
+
+# We should be installable with either setuptools or distutils.
+package_info = dict(
+    name='zulipbots',
```

Comment:
I know I am not sure what we should name it yet. I think I'll wait on feedback from everyone else and see what's the general consensus on name choices. :) Thanks for pointing this out!